uvloop/handles/process.pyx

```diff
@@ -115,17 +117,8 @@ cdef class UVProcess(UVHandle):
                         break
 
         finally:
-            os_close(self._errpipe_read)
-            try:
-                os_close(self._errpipe_write)
-            except OSError:
-                # Might be already closed
-                pass
-
-            fds_to_close = self._fds_to_close
-            self._fds_to_close = None
```

Comment:
I think `self._fds_to_close = None` is still needed?

Reply:
Ah yep I see it

Reply:
I moved it up so that _close_after_spawn can't interfere with the errpipe_write fd